Makefile.include

```diff
@@ -42,6 +42,10 @@ BINDIRBASE := $(abspath $(BINDIRBASE))
 BINDIR ?= $(BINDIRBASE)/$(BOARD)
 BINDIR := $(abspath $(BINDIR))/
 
+ifneq (9, $(words ${RIOTBASE} ${CCACHE_BASEDIR} ${RIOTCPU} ${RIOTBOARD} ${RIOTPROJECT} ${RIOTPKG} ${APPDIR} ${BINDIRBASE} ${BINDIR}))
```

Comment:
please add a substitution for each variable in case it is undefined or empty:
```
ifneq (9, $(words ${RIOTBASE}:-default ${CCACHE_BASEDIR}:-default ${RIOTCPU}:-default ${RIOTBOARD}:-default ${RIOTPROJECT}:-default ${RIOTPKG}:-default ${APPDIR}:-default ${BINDIRBASE}:-default ${BINDIR}:-default))
```